[#7015] Fix amount validation in chat input

speedyFixer opened the pull request:
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)


fixes #7015 

### Summary

[comment]: # (Summarise the problem and how the pull request solves it)
A code is deleting (Cleaning) the validation errors, this occurs too quickly, so users don't be able to see any validation error and the text in the input chat is cleared after It. So, I added a validation and changed the style of the tooltip and send button as the [figma](https://www.figma.com/file/aS1ct66VQ6V0cio7vSqS8UoG/Chat?node-id=144%3A234) file describe.

##### Functional

- 1-1 chats


### Steps to test
<!-- (Specify exact steps to test if there are such) -->
As the issue #7015 describe.

<!-- (PRs will only be accepted if squashed into single commit.) -->

status: ready <!-- Can be ready or wip -->

speedyFixer committed:
[#7015] Fix amount validation in chat input

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :x: | 061b37e8 | [#3](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-8537/3/) | 2019-07-08 14:25:04 | ~3 min | `ios` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-8537/3/consoleText) |

speedyFixer commented:
I can't see the log of failed build on iOS, Any hint?

flexsurfer commented:
@speedyFixer please try to rebase onto develop, thanks

speedyFixer commented:
@flexsurfer Done!

speedyFixer commented:
I saw the failed test, but what is the PlusButton? Thanks

rachelhamlin commented:
@flexsurfer @yenda can you guys review this one once anything release related is out of the way? Does not need to be included in v0.14.

flexsurfer commented:
@errorists could you please take a look? does it work as expected? thanks

speedyFixer commented:
@flexsurfer Updated with the latest changes in develop, but I can't see the error in Jenkins about the iOS. Any hint?

speedyFixer commented:
If anyone can help me showing me the iOS error in the CI I can fix it. Thanks

yenda commented:
@speedyFixer it is related to nix, to `git pull --rebase origin develop` and it should fix it

speedyFixer commented:
Thanks @yenda , I'm going to fix it.

speedyFixer commented:
@yenda I updated the branch with the latest changes from develop, but the issue was shown again.

yenda commented:
@speedyFixer it was just a timeout I restarted the build

speedyFixer commented:
@yenda Thanks!

oskarth commented:
What's the state of this @speedyFixer @yenda?

flexsurfer commented:
we're waiting for feedback from design team first,  cc @errorists

flexsurfer commented:
sorry @speedyFixer could you please rebase onto develop thanks

hesterbruikman commented:
@speedyFixer Thanks for your patience! Not seeing the changes you made on Android:/ When I enter an invalid amount at the moment I don't get any tooltip and the arrow remains blue. Maybe rebase to develop fixes this?

Please ping once's you;ve gotten around to that so I can check again:)

cc @errorists

speedyFixer commented:
Hi @flexsurfer done, I'm getting errors in my local environment, Gradle 5.1.1 throws android SDK path doesn't exist (It exists), so I could not build status-react, therefore I just did a rebase and pushed changes, (I couldn't do manual testing).  

@flexsurfer Is the android CI error a timeout? or is something more?

@hesterbruikman I think you can review again.

speedyFixer commented:
@flexsurfer , with the last status-react version I see this issue:

![error_status](https://user-images.githubusercontent.com/46165430/62473594-ac7e0d00-b766-11e9-970a-c256f3dad436.png)

I was unable to login and test if the fix for this issue still works with the rebase result.

speedyFixer commented:
I fixed adding the missing file for US translations and I added again my account. 

This PR doesn't work anymore, seems like some changes in the chat cause that commands don't be recognized. Any clue? @flexsurfer @hesterbruikman 

This fix was working, maybe some new commit broke this.

speedyFixer commented:
Some hint?

hesterbruikman commented:
Hi @speedyFixer, unfortunately it makes sense that the PR no longer works...

This issue crossed with a more recent decision and PR to temporarily remove command line transactions in chat altogether. 

This is a temporary situation as we definitely want to bring transactions in chat back (as CLI or new GUI). Reason is that we will be introducing the concept of multiple accounts/addresses in the near future. With this feature we also need to add a proof-of-address mechanism between clients to ensure secure chat transactions.

I will close this issue as it can take up to months before we can revisit, at which point we can reopen.

Thanks so much for your time working on this. Very sorry these changes impact your efforts.

speedyFixer commented:
@hesterbruikman thanks for your clarification, I'm a bit disappointed , because my work is unusable. I hope help more in the status project.
